css/style.css

```diff
@@ -38,22 +38,201 @@ Developer @GA in NYC
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
 
+
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
 body {
+  font-family: OpenSans;
+  font-size: 24px;
+  font-weight: 300;
+  color: #606161;
+  background-color: #f0efef;
+  line-height: 32px;
+}
+
+header {
+  background-color: #033048;
+  padding: 30px 0;
+}
+
+h1 {
+  font-family: OpenSans;
```

Comment:
Once you define 'Open Sans' in the `html` selector, the font will inherit to the child elements. Therefore, you can omit the font-family rule everywhere but the `html`